### DISCUSSION TITLE
Enhancement: Week Option suggestion

### DISCUSSION THREAD

**Discussion by SpringyToast** [FunkinCrew/Funkin#1665] [Ideas]
In the new Week 7 update two new options were added that were pretty neat, the only issue I have is that it clutters up the pause menu when I pause the game. I suggest that there should be a new option added called "Week options" that would be sort of like the options menu that we have but with some differences for the new rules in matches. This would help clean up the pause menu and allow more content to be added to the pause menu if wanted.

**Comment by GOOBOZA** [comment]
i agre
